client/components/suggestions/index.jsx

```diff
@@ -227,10 +254,20 @@ const Suggestions = React.createClass( {
 				<div className="suggestions__category" key={ key }>
 					<span className="suggestions__category-name">{ key }</span>
 					<span className="suggestions__category-counter">
-						{ this.translate( '%(filtered)s of %(total)s', {
+						{ i18n.translate( '%(filtered)s of %(total)s', {
 							args: { filtered, total }
 						} ) }
 					</span>
+					{ this.props.terms[ key ].length > suggestions[ key ].length &&
+						<ShowAll
+							onClick={ this.onShowAllClick }
+							cat={ key }
+							txt={ i18n.translate( 'Show All' ) } /> }
+					{ key === this.state.showAll &&
+						<ShowAll
+							onClick={ this.onShowAllClick }
+							cat={ '' }
+							txt={ i18n.translate( 'Show Less' ) } /> }
```

Comment:
Hi! I've found a possible matching string that has already been translated **27** times:
`translate( 'Show less' )` *ES Score: 13*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).